src/plugins/workspace/public/components/workspace_overview/workspace_overview.tsx

```diff
@@ -0,0 +1,207 @@
+
+export const IS_WORKSPACE_OVERVIEW_COLLAPSED_KEY = 'workspace:overview_collapsed';
+
+export interface WorkspaceOverviewProps {
+  workspaceConfigurableApps$?: BehaviorSubject<PublicAppInfo[]>;
+}
+
+export const WorkspaceOverview = (props: WorkspaceOverviewProps) => {
+  const {
+    services: { workspaces, application, http },
+  } = useOpenSearchDashboards<CoreStart>();
+
+  const currentWorkspace = useObservable(workspaces.currentWorkspace$);
+  const currentWorkspaceId = useObservable(workspaces.currentWorkspaceId$);
+
+  // workspace level setting
+  const workspaceOverviewCollapsedKey = `${IS_WORKSPACE_OVERVIEW_COLLAPSED_KEY}_${
+    currentWorkspaceId || ''
+  }`;
+
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [isGettingStartCardsCollapsed, setIsGettingStartCardsCollapsed] = useState(false);
+
+  useEffect(() => {
+    setIsGettingStartCardsCollapsed(localStorage.getItem(workspaceOverviewCollapsedKey) === 'true');
+  }, [workspaceOverviewCollapsedKey]);
+
+  /**
+   * all available cards based on workspace selected features
+   */
+  const availableCards = useMemo(() => {
+    const features = currentWorkspace?.features || ['*'];
+    const featureFilter = featureMatchesConfig(features);
+    return getStartCards.filter((card) => !card.appId || featureFilter({ id: card.appId }));
+  }, [currentWorkspace]);
+
+  if (!currentWorkspace) {
+    return null;
+  }
+
+  const pageTitle = (
+    <EuiFlexGroup gutterSize="none" alignItems="baseline" justifyContent="flexStart">
+      <EuiFlexItem grow={false}>{currentWorkspace?.name}</EuiFlexItem>
+    </EuiFlexGroup>
+  );
+
+  const tabs = [
+    {
+      id: 'overview',
+      name: i18n.translate('workspace.overview.tabTitle', {
+        defaultMessage: 'Overview',
+      }),
+      content: <WorkspaceOverviewContent />,
+    },
+    {
+      id: 'library',
+      name: i18n.translate('workspace.overview.library.tabTitle', {
+        defaultMessage: 'Library',
+      }),
+      content: <></>,
+    },
+    {
+      id: 'settings',
+      name: i18n.translate('workspace.overview.setting.tabTitle', {
+        defaultMessage: 'Settings',
+      }),
+      content: <WorkspaceOverviewSettings {...props} />,
+    },
+  ];
+
+  const collapseButton = (
+    <EuiButtonEmpty
+      color="text"
+      size="xs"
+      data-test-subj={isGettingStartCardsCollapsed ? 'Expand' : 'Collapse'}
+      aria-label={isGettingStartCardsCollapsed ? 'Expand' : 'Collapse'}
+      iconType={isGettingStartCardsCollapsed ? 'arrowDown' : 'arrowUp'}
+      iconSide="right"
+      onClick={() => {
+        const newValue = !isGettingStartCardsCollapsed;
+        setIsGettingStartCardsCollapsed(newValue);
+        localStorage.setItem(workspaceOverviewCollapsedKey, newValue ? 'true' : 'false');
+      }}
+    >
+      {isGettingStartCardsCollapsed ? 'Expand' : 'Collapse'}
+    </EuiButtonEmpty>
+  );
+
+  const rightSideItems: ReactNode[] = isGettingStartCardsCollapsed ? [collapseButton] : [];
```

Comment:
Does it means once user collapse the section, he/she won't be able to expand it any more?

Reply:
![image](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/113890546/35c247ad-96a5-45e7-8be6-a15832f8fca4)
There will have a `Expand` button on the top right

Reply:
Thanks for clarification.